### DISCUSSION TITLE
Services should be able to be accessed from other nodes than the one the pods are running on.

### DISCUSSION THREAD

**Discussion by dillfrescott** [k3s-io/k3s#9762] [Q&A]
This would be nice!! That way i wouldnt have to use port-forward and could use NodePort instead

**Comment by brandond** [comment]
You should be able to access clusterip services from any node in the cluster. What exactly are you trying, and what is the result?

**Reply by dillfrescott** [reply]
I am trying to set up a service that runs on two nodes, a vps and a laptop behind nat. I am still learning but it seems as though kubectl port forward works fine to connect to services running on other nodes but not node port. 

It may be a networking issue on my part the more I tinker around with the software. 

But it's odd that port forward works fine between the nodes but nodeport does not. How is it only working with one command and not the other?
